src/context/index.ts

```diff
@@ -161,19 +161,19 @@ export const setupContext = async (
       return new AuthenticationClient({
         domain: AUTH0_DOMAIN,
         clientId: AUTH0_CLIENT_ID,
-        clientAssertionSigningKey: readFileSync(AUTH0_CLIENT_SIGNING_KEY_PATH),
+        clientAssertionSigningKey: readFileSync(AUTH0_CLIENT_SIGNING_KEY_PATH, 'utf8'),
         clientAssertionSigningAlg: !!AUTH0_CLIENT_SIGNING_ALGORITHM
           ? AUTH0_CLIENT_SIGNING_ALGORITHM
           : undefined,
       });
     })();
 
-    const clientCredentials = await authClient.clientCredentialsGrant({
+    const clientCredentials = await authClient.oauth.clientCredentialsGrant({
       audience: config.AUTH0_AUDIENCE
         ? config.AUTH0_AUDIENCE
         : `https://${config.AUTH0_DOMAIN}/api/v2/`,
     });
-    return clientCredentials.access_token;
+    return clientCredentials.data?.access_token;
```

Comment:
Should introduce some checks to prevent an undefined `access_token` from being persisted through the rest of the process.

Reply:
**Recommendation:** Ensure ample amount of undefined checks where appropriate.